Source/WebCore/platform/unix/SharedMemoryUnix.cpp

```diff
@@ -76,11 +82,11 @@ static inline int accessModeMMap(SharedMemory::Protection protection)
     return PROT_READ | PROT_WRITE;
 }
 
-static UnixFileDescriptor createSharedMemory()
+static UnixFileDescriptor createSharedMemory([[maybe_unused]] size_t size)
 {
     int fileDescriptor = -1;
 
-#if HAVE(LINUX_MEMFD_H)
+#if HAVE(LINUX_MEMFD_H) && !HAVE(ASHAREDMEMORY)
```

Comment:
This file is small enough and already has several ifdefs, could we just add SharedMemoryAndroid.cpp?